examples/test_tesla_impact_report/test_gpt4o.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    splitter = SentenceSplitter()
</pre><p>Should we do a page splitter?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
we could write simple code in the notebook, but i'd prefer to have a module in the framework or the llamaparse client lib actually

e.g. `LlamaParsePageSplitter`